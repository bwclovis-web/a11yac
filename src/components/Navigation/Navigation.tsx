import React from 'react';
import { Link } from "gatsby"
import styled from 'styled-components';

const NavigationStyles = styled.nav`
    background-color: var(--grey);
    height: calc(100% - 6rem);
    position: sticky;
    height: calc(100vh - 94px);
    padding: 120px 100px;
`

const Navigation = () => (
    <NavigationStyles>
        <ul>
            <li>
                <Link to ="/new-component">New Component</Link>
            </li>
            <li><Link to ="/component-listing">Current Components</Link></li>
        </ul>
    </NavigationStyles>
)

export default Navigation;