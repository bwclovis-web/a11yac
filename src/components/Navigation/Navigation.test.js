import React from 'react';
import renderer from 'react-test-renderer';
import Navigation from './Navigation';

describe("Navigation", () => {
    it('should render correctly ', () => {
        const tree = renderer.create(<Navigation/>).toJSON()
        expect(tree).toMatchSnapshot()
    });
})