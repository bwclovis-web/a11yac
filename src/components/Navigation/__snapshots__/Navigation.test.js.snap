// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navigation should render correctly  1`] = `
<nav
  className="sc-bdfBwQ etJiqB"
>
  <ul>
    <li>
      <a
        href="/new-component"
        onClick={[Function]}
        onMouseEnter={[Function]}
      >
        New Component
      </a>
    </li>
    <li>
      <a
        href="/component-listing"
        onClick={[Function]}
        onMouseEnter={[Function]}
      >
        Current Components
      </a>
    </li>
  </ul>
</nav>
`;
